src/actor.js

```diff
@@ -764,3 +764,40 @@ export const getApifyProxyUrl = (options = {}) => {
 
     return `http://${username}:${password}@${hostname}:${port}`;
 };
+
+/**
+ *
+ * Creates adhoc webhook for current actor run.
+ * Webkooks are supported only for runs on Apify cloud.
+ *
+ * @param eventTypes {String[]} - Array of event types, which you can set for actor run, see
+ * the <a href="https://apify.com/docs/webhooks#events-actor-run" target="_blank">actor run events</a> in the Apify doc.
+ * @param requestUrl {String} - URL which will be requested using HTTP POST request, when actor run will be in specific event type.
+ *
+ * @return {Promise<Webhook|undefined>}
+ *
+ * @memberof module:Apify
+ * @function
+ * @name createAdHocWebhook
+ */
+export const createAdHocWebhook = async (eventTypes, requestUrl) => {
+    checkParamOrThrow(eventTypes, 'eventTypes', 'Array');
+    checkParamOrThrow(requestUrl, 'requestUrl', 'String');
+
+    const runId = process.env[ENV_VARS.ACTOR_RUN_ID];
+    if (!runId) {
+        log.warning('Webhooks are not supported locally! We skipped createAdHocWebhook() invocation.');
```

Comment:
"Apify.webhook() is only supported when running on the Apify platform. The webhook will not be invoked."

Reply:
I think that we should use here `isAtHome()` instead of this env var as the meaning of this `IS_AT_HOME` env var is exactly if the actor is running on Apify platform.

Reply:
So you'd check that and print warning, and throw error if run_id is not defined? I guess that's better.

BTW, I can imagine that in the future we'll add support for webhooks also for local runs.

Reply:
I created a new issue for that #345